Solutions/TenableIO/Data Connectors/TenableIO.json

```diff
@@ -0,0 +1,156 @@
+{
+	"id": "TenableIOAPI",
+	"title": "Tenable.io Vulnerability Management",
+	"publisher": "Tenable",
+	"descriptionMarkdown": "The [Tenable.io](https://www.tenable.com/products/tenable-io) data connector provides the capability to ingest Asset and Vulnerability data into Azure Sentinel through the REST API from the Tenable.io platform (Managed in the cloud). Refer to [API documentation](https://developer.tenable.com/reference) for more information. The connector provides the ability to get data which helps to examine potential security risks, get insight into your computing assets, diagnose configuration problems and more",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a [**Tenable.io parser**](https://aka.ms/sentinel-tenableio-parser) based on Kusto to work as expected which is deployed with the Azure Sentinel Solution.",
+	"graphQueries": [
+		{
+			"metricName": "Total data received",
+			"legend": "Tenable_IO_Assets_CL",
+			"baseQuery": "Tenable_IO_Assets_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "Tenable_IO_Vulns_CL",
+			"baseQuery": "Tenable_IO_Vulns_CL"
+		}
+	],
+	"sampleQueries": [
+		{
+			"description": "TenableIO VM Report - All Assets",
+			"query": "Tenable_IO_Assets_CL\n | sort by TimeGenerated desc"
+		},
+		{
+			"description": "TenableIO VM Report - All Vulns",
+			"query": "Tenable_IO_Vulns_CL\n | sort by TimeGenerated desc"
+		},
+		{
+			"description": "Select unique vulnerabilities by a specific asset.",
+			"query": "Tenable_IO_Vuln_CL\n | where asset_fqdn_s has \"one.one.one.one\"\n | summarize any(asset_fqdn_s, plugin_id_d, plugin_cve_s) by plugin_id_d"
+		},
+		{
+			"description": "Select all Azure assets.",
+			"query": "Tenable_IO_Assets_CL\n | where isnotempty(azure_resource_id_s) or isnotempty(azure_vm_id_g)"
+		}
+	],
+	"dataTypes": [
+		{
+			"name": "Tenable_IO_Assets_CL",
+			"lastDataReceivedQuery": "Tenable_IO_Assets_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "Tenable_IO_Vulns_CL",
+			"lastDataReceivedQuery": "Tenable_IO_Vulns_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"Tenable_IO_Assets_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		},
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"Tenable_IO_Vulns_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 1,
+		"isPreview": true
+	},
+	"permissions": {
+		"resourceProvider": [
+			{
+				"provider": "Microsoft.OperationalInsights/workspaces",
+				"permissionsDisplayText": "read and write permissions are required.",
```

Comment:
Please update "permissionsDisplayText" as per template to pass PR validations:
` "read and write permissions on the workspace are required." ` 
See [template link](https://github.com/Azure/Azure-Sentinel/blob/1137aaf2e692e4349f94dfdf3056f6f757ed361a/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/DataConnector_API_AzureFunctionApp_template.json#L41) for refernce